docs/graphql/manual/guides/postgres/indexes.rst

```diff
@@ -0,0 +1,47 @@
+.. meta::
+   :description: Use Postgres indexes with Hasura
+   :keywords: hasura, docs, postgres, indexes
+
+.. _postgres_indexes:
+
+Postgres indexes
+================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+`Postgres indexes <https://www.postgresql.org/docs/current/sql-createindex.html>`__ are a way of increasing performance on a field that is queried frequently. The concept is similar to the one of an index in a book. 
```

Comment:
In general, in the context of database, we should used columns instead of fields.